app/views/chat_channels/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <% title "DEV Connect 👩‍💻💬👨‍💻" %>
 <%= content_for :page_meta do %>
-  <link rel="canonical" href="https://dev.to/connect" />
+  <link rel="canonical" href="<%= app_url %>/connect" />
```

Comment:
you can pass the path as an argumento to `app_url`, so in this case it would be:

`app_url(connect_path)`

Reply:
oh my gawd this took forever 😅 but all the app_urls done now 😄Thanks for catching this Rhymes.